suite-native/biometrics/src/useBiometrics.tsx

```diff
@@ -19,19 +21,18 @@ export const useBiometrics = () => {
     const { isBiometricsOptionEnabled } = useIsBiometricsEnabled();
     const appState = useRef(AppState.currentState);
     const { isUserAuthenticated, setIsUserAuthenticated } = useIsUserAuthenticated();
+    const goneToBackgroundAt = useRef(Number.MAX_SAFE_INTEGER);
```

Comment:
I would prefer to set the default value to `null` rather than `Number.MAX_SAFE_INTEGER`
```suggestion
    const goneToBackgroundAt = useRef(null);
```

Then you could change line `28` as follows:
```jsx
if (!goneToBackgroundAt.current || goneToBackgroundAt.current > Date.now() - HALF_MINUTE) {
```

IMO the `Number.MAX_SAFE_INTEGER` value is counterintuitive and does not bring any value to this code. But maybe it is only my preference, won't block this PR by this.

Reply:
`if (!goneToBackgroundAt.current || goneToBackgroundAt.current > Date.now() - HALF_MINUTE) {`

This wouldn't make sense as on the first render it wouldn't authenticate the user at all. 

First evalutation would be `!goneToBackgroundAt.current` = !null = true and would fail on comparing null to Date. 

If it would be `if (!goneToBackgroundAt.current && goneToBackgroundAt.current > Date.now() - HALF_MINUTE) {` that would work but I think it's just adding more complexity and room for mistakes like this one that is not necessary. I could write a comment on why the default value is set for `Math.MAX_SAFE_INTEGER` if you think it's necessary. Or if @Nodonisko will agree it doesn't make sense, I'm fine with it, just my 2 cents.